docs/src/main/paradox/docs/delta/api/current/views/composite-view-api.md

```diff
@@ -249,259 +256,354 @@ The following example creates a Composite view containing 3 sources and 2 projec
-    "value": "10 minutes"
-  }  
+    "value": "10 minute"
+  }
 }
 ```
 
-## Endpoints
 
-The following sections describe the endpoints that are specific to a CompositeView.
-
-### Search Documents in a projection
+## Create using POST
 
 ```
-POST /v1/views/{org_label}/{project_label}/{view_id}/projections/{projection_id}/_search
+POST /v1/views/{org_label}/{project_label}
   {...}
 ```
 
-where `{projection_id}` is the @id value of the target `ElasticSearch` projection. 
+The json payload:
 
-The supported payload is defined on the 
-@link:[ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html){ open=new }
+- If the `@id` value is found on the payload, this `@id` will be used.
+- If the `@id` value is not found on the payload, an `@id` will be generated as follows: `base:{UUID}`. The `base` is the
+  `prefix` defined on the resource's project (`{project_label}`).
 
+**Example**
+
+Request
+:   @@snip [create.sh](../assets/views/composite/create.sh)
+
+Payload
+:   @@snip [payload.json](../assets/views/composite/payload.json)
+
+Response
+:   @@snip [created.json](../assets/views/composite/created.json)
+
+## Create using PUT
+
+This alternative endpoint to create a view is useful in case the json payload does not contain an `@id` but you want to
+specify one. The `@id` will be specified in the last segment of the endpoint URI.
+
+```
+PUT /v1/views/{org_label}/{project_label}/{view_id}
+  {...}
+```
+
+Note that if the payload contains an `@id` different from the `{view_id}`, the request will fail.
 
 **Example**
 
 Request
-:   @@snip [composite-view-es-id-search.sh](../assets/views/composite-view-es-id-search.sh)
+:   @@snip [create-put.sh](../assets/views/composite/create-put.sh)
 
 Payload
-:   @@snip [composite-view-es-payload.json](../assets/views/composite-view-es-search-payload.json)
+:   @@snip [payload.json](../assets/views/composite/payload.json)
 
 Response
-:   @@snip [composite-view-es-search.json](../assets/views/composite-view-es-search.json)
+:   @@snip [created-put.json](../assets/views/composite/created-put.json)
+
+## Update
+
+This operation overrides the payload.
 
-### Search Documents in all projections
+In order to ensure a client does not perform any changes to a resource without having had seen the previous revision of
+the view, the last revision needs to be passed as a query parameter.
 
 ```
-POST /v1/views/{org_label}/{project_label}/{view_id}/projections/_/_search
+PUT /v1/views/{org_label}/{project_label}/{view_id}?rev={previous_rev}
   {...}
 ```
 
-The supported payload is defined on the 
-@link:[ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html){ open=new }
+... where `{previous_rev}` is the last known revision number for the view.
+
+@@@ note { .warning }
 
+Updating a view creates a new Elasticsearch index and deletes the existing one. The indexing process will start from the
```

Comment:
It creates a new index/namespace for all the projection, not only for Elasticsearch.